### PR TITLE
javascript-if Timothy Lee

### PR DESCRIPTION
**Pull request by timlee712**
js-if-isUnderFive
js-if-isEven
js-if-startsWithJ

### CODE REVIEW

**Comment by timlee712** [comment]
js-if-categorizeAcidity
js-if-introduceWarnerBro

File: javascript-if/main.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return false;
+}
+
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'neutral';
+  } else if (pH < 7 && pH > 0) {
```
Comment (shawnkost) [inline javascript-if/main.js:58]: This should be greater than or equal to. `pH >= 0`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One small change Timothy!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**